code/style.css

```diff
@@ -1,7 +1,482 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+* {
+    box-sizing: border-box;
 }
+
+body{
+  margin: 0;
+  min-height: 100vh;
+ background: rgb(0,18,18);
+ background: linear-gradient(180deg, rgb(5 29 31) 46%, rgb(0 29 46 / 80%) 100%);
+}
+
+.hero-video-wrap{
+  position: sticky;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.hero-video{
+   background-size: cover;
+    height: auto;
+    object-fit: cover;
+    width: 100%;
+    max-height: 65vh;
+    filter: blur(0.5px) saturate(0.5);
+    z-index: -1;
+}
+
+.hero-normal-titles-wrapper{
+  z-index: 1;
+  position: absolute;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 3%;
+}
+
+.hero-title-normal{
+  font-size: 6em;
+  color: #c2b3a8;
+  font-family: 'Big Shoulders Display', cursive;
+  margin: 0;
+}
+
+.hero-small-title-normal{
+  font-size: 4em;
+  color: #c2b3a8;
+  font-family: Sacramento, cursive;
+  margin: 0;
+}
+
+.hero-mobile {
+  height: 40vh;
+  background-image: url("./pics/jungleView.jpg");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Big Shoulders Display', cursive;
+  margin-bottom: 4vh;
+  min-width: 300px;
+  display: none;
+}
+
+.hero-header-mobile{
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+#header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.hero-title-mobile{
+  background: #e8f3f8b8;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 2.8em;
+  margin-top: 0;
+}
+
+.hero-small-title-mobile{
+  background: #e8f3f8b8;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 2.5em;
+  margin:0;
+  font-family: Sacramento, cursive;
+}
+
+.hero-mobile, .signup-form-wrap, .accordion-Wrap{
+  padding: 20px;
+}
+
+.flex-container{
+  display: flex;
+  background-color: transparent;
+  justify-content: space-evenly;
+  align-items: stretch;
+  flex-direction : row;
+  flex-wrap: wrap;
+  padding-top: 3%;
```

Comment:
I see you really like to experiment with different units in paddings :D As before, `px` are the most proper ones for that